native/spark-expr/src/lib.rs

```diff
@@ -62,6 +61,8 @@ mod unbound;
 pub use unbound::UnboundColumn;
 pub mod utils;
 pub use normalize_nan::NormalizeNaNAndZero;
+mod predicate_funcs;
+pub use predicate_funcs::*;
```

Comment:
nit: can we make this explicit rather than use wildcard?

Reply:
done